xarray/core/accessor_str.py

```diff
@@ -682,6 +699,19 @@ def lower(self) -> T_DataArray:
         Returns
         -------
         lowerd : same type as values
```

Comment:
because I saw this while reviewing (this has been like that before this PR):
```suggestion
        lowered : same type as values
```

Reply:
corrected it!